megatron/optimizer/optimizer.py

```diff
@@ -51,6 +49,9 @@ def _multi_tensor_copy_this_to_that(this, that, overflow_buf=None):
     is not provided, we default back to simple loop copy to be compatible
     with bfloat16."""
     if overflow_buf:
+        from apex.multi_tensor_apply import multi_tensor_applier
```

Comment:
Similar to the gradient clipping change earlier. We need to provide both Python implementation and opportunity for accelerator-specific implementations.

Reply:
The python implementation already exists. We have adjusted the if condition.